### PR TITLE
cellL10n: Small adjustements

### PR DESCRIPTION
**Pull request by AniLeo**
Small adjustments on cellL10n.

- Partial implementation of jstrnchk;
- Replaces fmt::throw_exception() for cellL10n.todo() and return ConversionOK for String methods and 0 for character methods;
- SBCSstoUTF8s;

~List of games being tested and their functions:
NPEA00205 -> UCS2stoUTF8s 
BLES01501 -> SBCSstoUTF8s 
NPUB30065 -> SBCSstoUTF8s 
BLJS10256 -> jstrnchk [Partially implemented]~

### CODE REVIEW

**Comment by raven02** [comment]
from another game that massively use the l10n

E {PPU[0x1000000] Thread (main_thread) [0x00617d20]} PPU: 'jstrnchk' aborted
F {PPU[0x1000000] Thread (main_thread) [0x00617d20]} class std::runtime_error thrown: Unimplemented
(in file C:\rpcs3\rpcs3\Emu\Cell\Modules\cellL10n.cpp:390)

File: rpcs3/Emu/Cell/Modules/cellL10n.cpp [hunk 1]
```diff
@@ -26,122 +26,134 @@ bool _L10nCodeParse(s32 code, HostCode& retCode)
...
-	case L10N_UTF8:         retCode = 65001;        return false;
-	case L10N_UTF16:        retCode = 1200;         return false; // 1200=LE,1201=BE
-	case L10N_UTF32:        retCode = 12000;        return false; // 12000=LE,12001=BE
-	case L10N_UCS2:         retCode = 1200;         return false; // Not in OEM, but just the same as UTF16
-	case L10N_UCS4:         retCode = 12000;        return false; // Not in OEM, but just the same as UTF32
+	case L10N_UTF8:         	retCode = 65001;        return false;
+	case L10N_UTF16:        	retCode = 1200;         return false; // 1200=LE,1201=BE
+	case L10N_UTF32:        	retCode = 12000;        return false; // 12000=LE,12001=BE
+	case L10N_UCS2:         	retCode = 1200;         return false; // Not in OEM, but just the same as UTF16
+	case L10N_UCS4:        		retCode = 12000;        return false; // Not in OEM, but just the same as UTF32
```
Comment (danilaml) [inline rpcs3/Emu/Cell/Modules/cellL10n.cpp]: mixed tabs and spaces.

**Comment by AniLeo** [comment]
@danilaml I usually only tab stuff but doesn't look right on git, should one space all instead?

**Comment by danilaml** [comment]
@AniLeo Well, this whole file seems to be using spaces for alignment. Btw, why did you even need to reallign the code?

**Comment by AniLeo** [comment]
L10N_MUSIC_SHIFT_JIS didn't fit properly so needed to reallign. Will space everything then

**Pull request merged**